kedro/framework/session/session.py

```diff
@@ -112,12 +116,17 @@ def __init__(
         _register_hooks_setuptools(hook_manager, settings.DISABLE_HOOKS_FOR_PLUGINS)
         self._hook_manager = hook_manager
 
+        self._conf_source = conf_source or str(
+            self._project_path / settings.CONF_SOURCE
+        )
+
     @classmethod
     def create(  # pylint: disable=too-many-arguments
         cls,
         package_name: str = None,
         project_path: Union[Path, str] = None,
         save_on_close: bool = True,
+        conf_source: Optional[str] = None,
```

Comment:
```suggestion
        conf_source: Optional[str] = None,
        save_on_close: bool = True,
```

Also I think we might need to add it last in order to be strictly non-breaking change, because someone might actually rely on the ordering of the arguments (very unlikely, but still)... WDYT?

Reply:
I think you're right, I added it at the end of the function signature in [7f5281c](https://github.com/kedro-org/kedro/pull/2117/commits/7f5281c6fd411ac48f76c8537db7c0a947cfe43a).